### ISSUE TITLE
Excessive contrast

### ISSUE THREAD

**Issue by Shouldercannon** [EdgeTX/edgetx#3542]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### What part of EdgeTX is the focus of this bug?

Transmitter firmware

### Current Behavior

In the tab 4 of the main menu and telemetry, with an optimal contrast of 20, there is an overestimated contrast.
![20](https://user-images.githubusercontent.com/94894541/235213168-25e95e88-33b0-46c0-a935-e115657852aa.png)
The problematic menus with a contrast of 17 look fine.
![17](https://user-images.githubusercontent.com/94894541/235213455-71b8dfe1-51b4-47a9-8c8b-0e009b2f7258.png)

### Expected Behavior

The contrast should be the same for the entire menu.

### Steps To Reproduce

1. Push Sys and go to tab 4.
2. Push Tele.

### Version

2.8.2

### Transmitter

Radiomaster Boxer

### Operating System (OS)

Windows

### OS Version

Win10 x64 22H2

### Anything else?

This problem has been observed since version 2.8.1 and was inherited until version 2.8.3.

**Comment by ggoraa** [comment]
Damn, and I thought that my display was faulty 😅

**Comment by pfeerick** [comment]
I believe this is related to the number of items shown on the screen... if you add a telemetry sensor to the telemetry screen, or add a SF/GF, the contrast should lighten to match the rest of the menus.

**Comment by 3djc** [comment]
Contrast is set once and for all at start, and does not depend on screen content. This is due to the screen itself whre it's pwoer driver regulation is not what you would like it to be

**Comment by pfeerick** [comment]
That is not correct - at least not as far as the **_appearance_** of contrast changing dependent how much on-screen content there is. If you add one or two functions to say GF or SF screen, it will lighten to be the same as all the other screens. i.e. As if the screen updates faster when there is less content on it, and slower when there is more, rather than being locked to a consistent speed regardless of how much content there is onscreen.

**Comment by error414** [comment]
is any way how to fix it?
